pages/index.js

```diff
@@ -141,11 +167,16 @@ const Home = ({ latestStories }) => {
 
   const [caseData, setCaseData] = useState({})
   const [cumulativeCase, setCumulativeCase] = useState(null)
+  const [cumulativeTest, setCumulativeTest] = useState(null)
```

Comment:
is there any benefit to setting these to null initially compared to undefined? they're both falsy

Reply:
not sure, it was like this when i got it